pkg/validation/internal/community.go

```diff
@@ -24,6 +24,9 @@ const ocpLabelindex = "com.redhat.openshift.versions"
 // OCP version where the apis v1beta1 is no longer supported
 const ocpVerV1beta1Unsupported = "4.9"
 
+// olm.properties is a CSV annotation that defines the properties of a bundle.
+const olmproperties = "olm.properties"
```

Comment:
@tylerslaton this validator is deprecated we must do these changes in the https://github.com/redhat-openshift-ecosystem/ocp-olm-catalog-validator. This implementation was moved to this project because the max OCP version is OCP specific. 

We have tasks already asking for the pipelines and CI's do the replacement. Could you please give a hand here and push this one for the right project?

Reply:
It shows outdate based on; https://github.com/operator-framework/api/pull/217#pullrequestreview-860307771

Reply:
I just went ahead and reverted this change and added it into the CSV validator instead. This should avoid any changes to this deprecated file.